radiant/dags/sql/clinical/init/patient_access_create_table.sql

```diff
@@ -0,0 +1,13 @@
+CREATE TABLE IF NOT EXISTS {{ mapping.starrocks_patient_access }} (
+    'id' varchar(500) NULL COMMENT "",
+    'mrn' varchar(128) NULL COMMENT "",
+    'gender' varchar(128) NULL COMMENT "", 
+    'race' varchar(128) NULL COMMENT "",
+    'ethnicity' varchar(128) NULL COMMENT "",
+    'given_name' varchar(128) NULL COMMENT "",
+    'family_name' varchar(128) NULL COMMENT "",
+    'birth_date' varchar(128) NULL COMMENT "",
+    'deceased_boolean' varchar(128) NULL COMMENT "",
+    'address_postal_code' varchar(128) NULL COMMENT ""
+    ) ENGINE=OLAP
+   DUPLICATE KEY(`id`); --is this the right way to specify the field that acts as primary key? 
```

Comment:
If you want a real primary key, that allows to update a patient, then use PRIMARY KEY. 
Pros : you will never have two patients with same id
Cons : Insert will be slower